exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,11 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+    if (sentence.indexOf("code") > -1) {
```

Comment:
Another way is to use the function ``includes``

Can you please fix the indentation for this if/else statement?